src/Microsoft.Health.Fhir.CosmosDb/Features/Storage/FhirDataStore.cs

```diff
@@ -98,7 +98,7 @@ public sealed class FhirDataStore : IDataStore, IProvideCapability
                 _logger.LogDebug($"Upserting {resource.ResourceTypeName}/{resource.ResourceId}, ETag: \"{weakETag?.VersionId}\", AllowCreate: {allowCreate}, KeepHistory: {keepHistory}");
 
                 UpsertWithHistoryModel response = await _retryExceptionPolicyFactory.CreateRetryPolicy().ExecuteAsync(
-                    async () => await _upsertWithHistoryProc.Execute(
+                    async (ct) => await _upsertWithHistoryProc.Execute(
```

Comment:
Unused param. Either remove like below or look at adding/passing through the `CancellationToken` in the `Execute` function.
```suggestion
                    async _ => await _upsertWithHistoryProc.Execute(
```